src/conic_algorithm.jl

```diff
@@ -105,6 +106,8 @@ type PajaritoConicModel <: MathProgBase.AbstractConicModel
     # MIP constructed data
     model_mip::JuMP.Model       # JuMP MIP (outer approximation) model
     x_mip::Vector{JuMP.Variable} # JuMP vector of original variables
+    x_int::Vector{JuMP.Variable} # JuMP vector of integer variables
```

Comment:
Maybe call this a subvector to make clear that it's not distinct from ``x_mip``?